tinywallet/tinywallet/app.py

```diff
@@ -152,7 +152,8 @@ def setwallet(wallet):
 
     def shutdown(self, e=None):
         """Connected to the context menu "quit" option. Shut down TinyWallet."""
-        self.assetScreen.shutdown()
+        if hasattr(self, "assetScreen"):
+            self.assetScreen.shutdown()
```

Comment:
Let's define `self.assetScreen` in the constructor with an initial value of `None`. That's typically what I've done and I'm not sure why I wasn't doing that here.